minio/minio
# Use pointer based TLS field

## Description

This will help projects like `health-analyzer` to know when the tls
info is not available (TLS field would be nil)
Signed-off-by: Shireesh Anjal <shireesh@minio.io>

## Motivation and Context

This is required to ensure that `health-analyzer` can reliably perform TLS related checks,
and skip them when tls info is not available.

## How to test this PR?

- Build `minio` with this PR and start the server
- Build `mc` with PR [3866](https://github.com/minio/mc/pull/3866)
- Generate health report using `mc admin subnet health {alias} --airgap`
- Verify that the generated report contains the node `minio -> info -> tls -> tls_info_available` with value `true`

## Types of changes

## Checklist:
